package.json

```diff
@@ -40,7 +40,11 @@
     "ember-addon"
   ],
   "dependencies": {
+    "@glimmer/resolver": "^0.3.0",
     "babel-plugin-debug-macros": "^0.1.1",
+    "broccoli-babel-transpiler": "^5.6.2",
```

Comment:
Additionally `ember-cli` should be version `2.13.0-beta.2`. There is code in `index.js` which is run when the feature flag is enabled (which it should be for tests, but not for tests on older versions of Ember).